lib/modules/dosomething/dosomething_reportback/includes/ReportbackTransformer.php

```diff
@@ -105,20 +114,4 @@ protected function transform($reportback) {
     return $data;
   }
 
-
-  /**
-   * @param $ids
-   *
-   * @return array
-   */
-  protected function getItems($ids) {
-    $filters = array(
-      'fid' => $this->formatData($ids),
-    );
-
-    // Obtaining all Reportback items.
-    $query = dosomething_reportback_get_reportback_files_query_result($filters, 'all');
-
-    return services_resource_build_index_list($query, 'reportback-items', 'fid');
-  }
 }
```

Comment:
I've seeing these a lot, worth fixing. Github is complaining - best practice is to leave a blank line at the end of a file.